File: 1/solution.go
package main

import "fmt"

func cal(a int, b int) int {
	c := b / a;
	return a * c * (c+1) / 2;
}

func main() {
	fmt.Printf("%d\n", cal(3, 999) + cal(5, 999) - cal(15, 999));
}

File: 1/solution.cpp
#include <bits/stdc++.h>
using namespace std;
int cal(int a, int upper = 999) {
    int b = upper / a;
    return a * b * (b + 1) / 2;
}
int main() {
    printf("%d\n", cal(3) + cal(5) - cal(15));
    return 0;
}
